ingestion/functions/parsing/brazil_amapa/amapa.py

```diff
@@ -0,0 +1,167 @@
+
+        commorbidities_list = []
+        if "Diabetes" in raw_commorbidities:
+            commorbidities_list.append("diabetes mellitus")
+        if "Portador de doenças cromossômicas ou estado de fragilidade imunológica" in raw_commorbidities:
+            commorbidities_list.append("primary immunodeficiency disease or chromosomal disease")
+            #Is combined DO OK?
+        if "Gestante" in raw_commorbidities:
+            commorbidities_list.append("pregnancy") #Not in DO
+            #I have grouped 'gestante' and 'gestante de alto risco' (high-risk pregnancy) into the same 
+            #entry ("pregnancy") in the commorbidities list.
+        if "Doenças respiratórias crônicas descompensadas" in raw_commorbidities:
+            commorbidities_list.append("respiratory system disease")
+        if "Doenças renais crônicas em estágio avançado (graus 3, 4 ou 5)" in raw_commorbidities:
+            commorbidities_list.append("chronic kidney disease") 
+        if "Doenças cardíacas crônicas" in raw_commorbidities:
+            commorbidities_list.append("heart disease")
+
+        preexistingConditions["values"] = commorbidities_list
+        return preexistingConditions
+    return None
+
+def convert_location(raw_entry: str):
+    if raw_entry:
+        municipality = raw_entry
+    else:
+        municipality = ""
+    query_terms = (municipality,) + ("Amapá", "Brazil")
+    return {"query":  ", ".join(query_terms)}
+
+def convert_notes(raw_commorbidities:str, raw_notes_neighbourhood: str, raw_notes_indigenousEthnicity: str):
+    raw_notes = []
+    if "Imunossupressão" in raw_commorbidities:
+        raw_notes.append("Patient with immunosupression")
+        """ 
+        Immunosupression is not necessarily a disease as defined in the Disease Ontology 
+        (i.e. could be drug-induced, or the result of many diseases) so I have put this in notes instead
+        or pre-existing conditions.
+        """
+    if raw_notes_neighbourhood:
+        raw_notes.append("Neighbourhood: " + raw_notes_neighbourhood)
+    if raw_notes_indigenousEthnicity:
+        raw_notes.append("Indigenous ethnicity: " + raw_notes_indigenousEthnicity)
+
+    notes = (', ').join(raw_notes)
+    return notes
+
+def parse_cases(raw_data_file: str, source_id: str, source_url: str):
+    """Parses G.h-format case data from raw API data.
+        Caveats:
+        1. There are no patient ID/case ID in the raw API so we aren't able
+           to dedupe.
+        2. There are two files in the source for Amapa, one for confirmed cases and one for confirmed
+           deaths; some of these cases may also be deaths but without patient IDs we are unable to confirm.
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.reader(f)
+        next(reader) # Skip the header.
+        cases = []
+        for row in reader:
+            if float(row[_AGE_INDEX]) > 120: #We have entries as high as 351 - unclear if this is days.
```

Comment:
that's really fishy, they don't have a data dictionary explaining their data format? we should clarify that

Reply:
Not that I could find, no.